kubeportal/api/views.py

```diff
@@ -70,7 +66,7 @@ class WebApplicationViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = WebApplicationSerializer
 
     def get_queryset(self):
-        if 'user_pk' in self.kwargs:
+        if 'user_pk' in self.kwargs and self.kwargs['user_pk'] != 'null':
```

Comment:
Is this some hack specifically for your JS-code?

Reply:
well yes, I got an internal server error in some cases when the js frontend fucked up and requested e.g. /users/null/webapps . Just thought it could be an improvement to handle this. I'll have a look...